docs/testplan/multi_tier_latency_test.md

```diff
@@ -0,0 +1,28 @@
+# SONiC Switch Latency Test
+
+## Test Objective
+
+This test aims to measure the latency introduced by a switch under the fully loaded condition.
+
+## Test Setup
+
+The test is designed to be topology-agnostic, meaning it does not assume or impose a specific network connection. The only requirement is that the DUT is fully connected to handle full traffic loads under stress. Plus all traffic generators in the testbed must be time synchronized.
+
+## Background
+
+In Broadcom architecture, the Memory Management Unit (MMU) handles packet buffering and traffic management within the device. Each MMU consists of two Ingress Traffic Managers (ITMs). The first and last quarters of the ports reside in one ITM, while the second and third quarters reside in the other ITM. We will test the latency of traffic managed within a single ITM as well as traffic that traverses between the two ITMs.
+
+Cut-through packet forwarding mode is unsupported in SONiC. Store-and-forward is supported, but it can only be set in SAI. Config_db support is not in place. So no packet forwarding requirement in latency test.
```

Comment:
> Cut-through packet forwarding mode is unsupported in SONiC. Store-and-forward is supported, but it can only be set in SAI.

this is not true. we can just say, currently, we are testing with store-and-forward mode only.